src/OrbitVulkanLayer/SubmissionTracker.h

```diff
@@ -652,6 +652,8 @@ class SubmissionTracker : public VulkanLayerProducer::CaptureStatusListener {
     VkQueryPool query_pool = timer_query_pool_->GetQueryPool(device);
 
     if (!timer_query_pool_->NextReadyQuerySlot(device, slot_index)) {
+      // TODO(b/192998580): Send an appropriate CaptureEvent to notify the client that the Vulkan
+      //  layer was out of query slots in a certain time range.
       return false;
```

Comment:
Should we also have an `ERROR_ONCE` here?

Reply:
The issue is, that this will not show up in the Orbit log, but only in the game log. Not sure if we want that.

Reply:
Ah right, let's leave it as it is.